### PR TITLE
Launch tidy

### PR DESCRIPTION
**Pull request by ad-prince**
There are a lot of small changes here. In all articles I have tried to:
- Add and tidy links
- Ensure all sessions are local with relevant session names
- Review R code (e.g. ensure that <- rather than = hasn't sneaked in)
- Change heading levels for consistency

Articles themselves are mostly unchanged aside from obvious typos or where some extra content is needed for a link.

I would approach this review in a risk based manner. As this is a tidy up with little actual code or content changing it is fine to review a sample and check that they comply with the list above.

### CODE REVIEW

File: ons-spark/pyspark-intro/pyspark-intro.ipynb [hunk 1]
```diff
@@ -1491,43 +1701,36 @@
...
+    "- [`.to_csv()`](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_csv.html) \n",
+    "    \n",
+    "Other Links:\n",
+    "- [Databricks: Learning Spark](https://pages.databricks.com/rs/094-YMS-629/images/LearningSpark2.0.pdf)\n",
     "- [Wikipedia; Join (SQL)](https://en.wikipedia.org/wiki/Join_(SQL))\n",
-    "- [Databricks: Learning Spark](https://pages.databricks.com/rs/094-YMS-629/images/LearningSpark2.0.pdf)"
+    "\n",
+    "#### Acknowledgments\n",
+    "\n",
+    "Thanks to Karina Marks, Chris Musselle and Dave Beech for creating the initial version of this article."
```
Comment (robertswh) [inline ons-spark/pyspark-intro/pyspark-intro.ipynb:1733]: Did Dave Beech contribute towards this?

**Review by robertswh — COMMENTED** [review]
1 comment, no need for a change. 1 question just to make sure it is correct

File: ons-spark/_toc.yml [hunk 2]
```diff
@@ -16,26 +16,26 @@ parts:
   - file: pyspark-intro/f-col
```
[the remark this answers is not in the document]
Comment (ad-prince) [reply ons-spark/_toc.yml:18]: Really good point. No opinion either way so happy to stick with what we have.

File: ons-spark/pyspark-intro/pyspark-intro.ipynb (hunk 1 above)
Comment (ad-prince) [reply ons-spark/pyspark-intro/pyspark-intro.ipynb:1733]: When I looked at the previous commits he had a small one to fix some typos.

**Pull request merged**